sdcm/nemesis.py

```diff
@@ -1710,6 +1710,93 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
         finally:
             self.target_node.remoter.run("sudo /sbin/ifup eth1")
 
+    def streaming_err_and_rebuild(self, task='decommission'):
+        """
+        Stop streaming task in middle and rebuild the data on the node.
+        """
+
+        def streaming_task_thread(nodetool_task='decommission'):
+            """
+            Execute some nodetool command to start persistent streaming
+            task: decommission | rebuild | repair
+            """
+            try:
+                self.target_node.run_nodetool(nodetool_task)
+            except Exception:  # pylint: disable=broad-except
+                self.log.debug('%s is stopped' % nodetool_task)
+
+        streaming_thread = threading.Thread(target=streaming_task_thread, kwargs={'nodetool_task': task})
```

Comment:
It's not enough, we need to do something like CorruptAndRebuild and CorreuptAnd Reapir in order to assure that there were meaningful streamings.

Reply:
changed to call _destroy_data_and_restart_scylla() before repair or rebuild.